lib/features/vpc/data/vpc_notice.dart

```diff
@@ -0,0 +1,46 @@
+Estimate our audience size and usage patterns.
+Store information about the child's preferences, allowing us to customize the content according to individual interests.
+We use geolocation information we collect to determine whether the user is in a location where it’s possible to use the Application for donating. 
+Our Practices for Disclosing Children's Information
+We may disclose aggregated information about many of our users, and information that does not identify any individual or device. In addition, we may disclose children's personal information:
+To third parties we use to support the internal operations of our Application.
+If we are required to do so by law or legal process, such as to comply with any court order or subpoena or to respond to any government or regulatory request.
+If we believe disclosure is necessary or appropriate to protect the rights, property, or safety of Givt, our customers or others, including to:
+protect the safety of a child;
+protect the safety and security of the Application; or
+enable us to take precautions against liability.
+To law enforcement agencies or for an investigation related to public safety. 
+If Givt is involved in a merger, divestiture, restructuring, reorganization, dissolution, or other sale or transfer of some or all of Givt's assets, whether as a going concern or as part of bankruptcy, liquidation, or similar proceeding or event, we may transfer the personal information we have collected or maintain to the buyer or other successor. 
+Social Features 
+The Application allows parents to view information about their child’s donation activities and any good deeds that the child records, and parents may provide certain responses to this information. 
+Accessing and Correcting Your Child's Personal Information
+At any time, you may review the child's personal information maintained by us, require us to correct or delete the personal information, and/or refuse to permit us from further collecting or using the child's information.  
+You can review, change, or delete your child's personal information by:
+Logging into your account and accessing the profile page relating to your child.
+Sending us an email at support@givt.app. To protect your and your child’s privacy and security, we may require you to take certain steps or provide additional information to verify your identity before we provide any information or make corrections. 
+Operators That Collect or Maintain Information from Children
+Givt Inc. is the operator that collects and maintains personal information from children through the Application. Givt can be contacted at support@givt.app, by mail at 3343 Peachtree Rd NE 145-1032, Atlanta, GA 30326, or by phone at 404-999-7024.
+""";
+}
```

Comment:
@pavelpavelko Can we move this into POEditor?

Reply:
done